lightning-tag/data-collection-control.md

```diff
@@ -0,0 +1,99 @@
+# Controlling Data Collection Timing
+
+Some Lotame clients do not want Lotame's data collection and targeting to run automatically. A use-case is checking for customer consent to tracking to comply with GPDR. In this use-case, the client does not want Lotame Lightning Tag to run until the customer has gone through consent management flow.
+
+This page has example javascript code to show the following flow:
+
+1. Webpage loads.
+1. Lotame Lightning Tag initializes but is configured to not run automatically by disabling autoRun.
```

Comment:
I assume this auto-renumbers itself on display ?

Reply:
Yup. Feature of Markdown. Allow simple re-ordering of your numbered list without having to re-type out numbers.